src/fig_rgb.f90

```diff
@@ -11,6 +11,7 @@ module fig_rgb
     end type RGB
 
     type(RGB) :: RED = RGB(255, 0, 0, 255) 
+    type(RGB) :: BLANK = RGB(0, 0, 0, 0) 
```

Comment:
To use these color wrappers in initialization expressions, you have to make them `parameters`, (similar to `constexpr` in c++): 

```
type(RGB), parameter :: BLANK = RGB(0,0,0,0)
```

Reply:
I will remove them soon and mainly use #14 for colors.